pandas/tests/indexes/interval/test_interval.py

```diff
@@ -544,6 +552,16 @@ def test_get_indexer_subintervals(self):
         expected = np.array([0, 0, 0], dtype='intp')
         tm.assert_numpy_array_equal(actual, expected)
 
+    # Make consistent with test_interval_new.py (see #16316, #16386)
```

Comment:
if this is consistent with those tests, you can remove from there I think (not sure exactly how we are tracking 'new' things vs current)

Reply:
k that is fine. yeah converting to use the new tests is a major effort :>